src/astx/callables.py

```diff
@@ -386,3 +386,32 @@ def get_struct(self, simplified: bool = False) -> ReprStruct:
         key = "YIELD-EXPR"
         value = {} if self.value is None else self.value.get_struct(simplified)
         return self._prepare_struct(key, value, simplified)
+
+
+@public
+@typechecked
+class YieldFromExpr(Expr):
+    """AST class for YieldFromExpr."""
+
+    value: Optional[Expr]
+
+    def __init__(
+        self,
+        value: Optional[Expr],
```

Comment:
```suggestion
        value: Expr,
```

in the `yield from`, the value is required:

```python

>>> def f():
...     yield from
  File "<stdin>", line 2
    yield from
              ^
SyntaxError: invalid syntax
>>> 
```